lib/net/ldap.rb

```diff
@@ -309,7 +309,14 @@ class Net::LDAP
     :constructed => constructed,
   }
 
+  universal = {
```

Comment:
@mtodd I have moved the value into universal, context specific constructed will not work because it starts at 160 and it's not a primitive.

To be honest I don't know why this stuff is written the way it is when all that is resulted is an array with 256 length with values at index with the types, why not just one simple flat dict.